frontstage/views/surveys/surveys_list.py

```diff
@@ -28,7 +29,9 @@ def get_survey_list(session, tag):
     if tag == 'todo':
         response = make_response(render_template('surveys/surveys-todo.html',
                                                  sorted_surveys_list=sorted_survey_list,
-                                                 added_survey=True if business_id and survey_id else None))
+                                                 added_survey=True if business_id and survey_id and not already_enrolled else None,
```

Comment:
This line is starting to get a bit long now.  Either put on the line above `added_survey = True if business_id and survey_id and not already_enrolled else None` (or something to that effect), or you could maybe roll this into a function? (something like `added_survey = is_survey_added()`on this line or something like that).
I mention rolling this into a function because this added survey is taking multiple bits of data to come to its answer, but it's not clear at a glance what each bit means.  That could also be me not understanding fully what's going on